nosetimer/plugin.py

```diff
@@ -311,6 +317,13 @@ def options(self, parser, env=os.environ):
         parser.add_option("--timer-warning", action="store", default=3,
                           dest="timer_warning", help=_warning_help)
 
+        # enforce timer
+        _enforce_timer_help = ("Fail tests that exceed the normal execution "
+                               "time (specified with --timer-ok)")
+        parser.add_option("--enforce-timer", action="store_true",
```

Comment:
This is a good idea. But does it make sense to allow the user to specify multiple values? I think that `--timer-enforce warning,error` would just do the same thing as `--timer-enforce warning` (since the "warning" threshold should be expected to be lower than the "error" threshold).

Reply:
@acordiner, I think it can be either way as long as it is clearly specified in README ;)